ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveMaterializedViewsRegistry.java

```diff
@@ -204,11 +205,21 @@ public void run() {
         } else {
           LOG.error("Problem connecting to the metastore when initializing the view registry", e);
         }
+      } finally {
+          cleanUpResources(ss);
```

Comment:
I'm curious about what would happen if we closed `ss` using `SessionState#close`.